src/main/java/seedu/address/model/reminder/ReminderList.java

```diff
@@ -65,43 +65,15 @@ public void add(Reminder reminder) {
     /**
      * Deletes a Reminder object from the current list.
      *
-     * @param reminder to be deleted.
+     * @param index of the {@code Reminder}to be deleted.
      * @return the updated ReminderList
      */
-    public ReminderList delete(Reminder reminder) {
+    public ReminderList delete(Index index) {
```

Comment:
`ReminderList` should not be mutable. You can return a new `ReminderList` with the `reminder` removed though.